src/Interpolations.jl

```diff
@@ -28,11 +28,12 @@ using WoodburyMatrices, Ratios, AxisAlgorithms
 import Base: convert, size, getindex, gradient, promote_rule
 
 abstract InterpolationType
+immutable NoInterp <: InterpolationType end
 abstract GridType
 immutable OnGrid <: GridType end
 immutable OnCell <: GridType end
 
-typealias DimSpec{T} Union(T,Tuple{Vararg{T}})
+typealias DimSpec{T} Union(T,Tuple{Vararg{Union(T,NoInterp)}},NoInterp)
```

Comment:
Then I guess I'll notice next time I'm on my Linux laptop. Working on Windows at the moment, so building from source is too much of a hassle...